src/main/java/com/noticemedan/mappr/viewmodel/MenuBarController.java

```diff
@@ -76,11 +75,9 @@ private void toggleDijkstra() {
 	}
 
 	private void toggleShortestPath() {
-		this.showShortestPath = !this.showShortestPath;
-		String labelStart = (this.showShortestPath) ? "Fjern" : "Vis";
 		MainViewController.getCanvas().toggleRandomShortestPath();
 		MainViewController.getCanvas().repaint();
-		showShortestPathMenuItem.setText(labelStart + " shortest path");
+		showShortestPathMenuItem.setText("Toggle shortest path");
```

Comment:
Hvad er argumentet for at fjerne "fjern / vis" og sætte "Toggle" i stedet?
Resten af programmets sprog i GUI er på dansk, i øvrigt.

Reply:
Hov skulle heller ikke være på engelsk, selvom shortest path vel sådan set også er det 🤔.. Men grunden til at det kun skulle være et ord er for at dække for edge cases, hvor man begynder at bruge det, mens der allerede er en path